docs/methoddocs/plugins.md

```diff
@@ -69,3 +69,111 @@
     :members:
     :show-inheritance:
 ```
+
+## Multicall
+
+Ape comes with builtin `multicall` method. Yes you're reading this right. No need to install external modules to do multicall, ape got you covered.Perform the Multicall call. This call will trigger again every time the `Call` object is called.
```

Comment:
Most of the words in the sentence are superfluous. I think only the first one (slightly re-written) is ok here.

```
The `ape-ethereum` core plugin comes with a multi-call module.
```

In general, docs should be simple and informative and not have any marketing terms.